themes/default/monster/view.php

```diff
@@ -2,7 +2,7 @@
 <h2>Viewing Monster</h2>
 <?php if ($monster): ?>
 <h3>
-	#<?php echo $monster->monster_id ?>: <?php echo htmlspecialchars($monster->iro_name) ?>
+	#<?php echo $monster->monster_id ?>: <?php echo htmlspecialchars($monster->name_japanese) ?>
```

Comment:
I dont get it, everywhere replaced name_japanese to name_english, but here you replace iro_name with name_japanese.


Shouldn't be iro_name => name_english and kro => name_japanese?

Reply:
I mean whole `themes/default/monster/view.php` file, not only these lines.